Source/d_main.c

```diff
@@ -1930,9 +1941,6 @@ void D_DoomMain(void)
   puts("\nP_Init: Init Playloop state.");
   P_Init();
 
-  puts("I_Init: Setting up machine state.");
```

Comment:
Why is it required to change the order here?

Reply:
This is because of the timer initialization. Actually, I'm not sure myself, but Chocolate Doom does it, and there is a desync with the old order.